### PR TITLE
Feature/fix responsiveness

### PR DESCRIPTION
**Pull request by xosnrdev**
## Description 

- Fixed landing page responsiveness and font.

### CODE REVIEW

File: app/page.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 import Footer from "./components/Footer_comp";
+import Hero from "./components/Hero";
+import HowItWorks from "./components/howItWorks_comp";
+import Testimonials from "./components/testimonials_comp";
 
 export default function Home() {
   return (
     <main className="min-h-screen flex justify-center overflow-x-hidden">
-      <div className="max-w-screen-2xl">
+      <div className="w-full">
```
Comment (Remi-dee) [inline app/page.js]: Please revert this change, we are not supporting any width larger than 1500px

**Review by Remi-dee — CHANGES_REQUESTED** [review]

File: app/page.js (hunk 1 above)
Comment (xosnrdev) [reply app/page.js]: On my of the bg image is not meeting both end 

Can i have a visual reference of what it looks like from your end when max-w-screen-2xl is used and when w-full is used?

Comment (xosnrdev) [reply app/page.js]: It's a Mac pc we could make it max-w-3xl or so what do you think ?

**Review by Remi-dee — APPROVED** [review]

**Review by Remi-dee — APPROVED** [review]
LG

**Pull request merged**